src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentStatusRepositorySpec.scala

```diff
@@ -68,4 +120,44 @@ class ConsignmentStatusRepositorySpec extends AnyFlatSpec with TestDatabase with
     consignmentStatusRetrieved.statustype should be(statusType)
     consignmentStatusRetrieved.modifieddatetime.get should be(modifiedTimestamp)
   }
+
+  "updateConsignmentStatus" should "only update the value of the status type passed in" in {
+    val db = DbConnection.db
+    val consignmentStatusRepository = new ConsignmentStatusRepository(db)
+    val consignmentId = UUID.fromString("2e998acd-6e87-4437-92a4-e4267194fe38")
+    val userId = UUID.fromString("7f7be445-9879-4514-8a3e-523cb9d9a188")
+    val statusTypeOne = "TransferAgreement"
+    val statusTypeTwo = "Upload"
+    val statusTypeThree = "Export"
+    val statusValueOne = "Completed"
+    val statusValueTwo = "Completed"
```

Comment:
Should statues one and two be something other than Completed? Otherwise this test can't tell whether the update method has changed them all to "Completed".

Reply:
Ah yes! Thanks, I will change this